<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DlgAskPassphrase</class>
 <widget class="QDialog" name="DlgAskPassphrase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>399</width>
    <height>204</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DlgEncryptWallet</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="m_lWarning">
     <property name="text">
      <string notr="true">Placeholder text</string>
     </property>
     <property name="textFormat">
      <enum>Qt::RichText</enum>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QFormLayout" name="m_lLayout">
     <property name="sizeConstraint">
      <enum>QLayout::SetMinimumSize</enum>
     </property>
     <property name="fieldGrowthPolicy">
      <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="m_lPass1">
       <property name="text">
        <string>Enter passphrase</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="m_lePass1">
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="m_lPass2">
       <property name="text">
        <string>New passphrase</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="m_lePass2">
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="m_lPass3">
       <property name="text">
        <string>Repeat new passphrase</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="m_lePass3">
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLabel" name="m_lCaps">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QCheckBox" name="m_cbAnonymization">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="visible">
        <bool>false</bool>
       </property>
       <property name="toolTip">
        <string>Serves to disable the trivial sendmoney when OS account compromised. Provides no real security.</string>
       </property>
       <property name="text">
        <string>For anonymization, automint, and staking only</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="m_bbOkCancel">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>m_bbOkCancel</sender>
   <signal>rejected()</signal>
   <receiver>DlgAskPassphrase</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>361</x>
     <y>180</y>
    </hint>
    <hint type="destinationlabel">
     <x>374</x>
     <y>129</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_bbOkCancel</sender>
   <signal>accepted()</signal>
   <receiver>DlgAskPassphrase</receiver>
   <slot>onAccept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>301</x>
     <y>179</y>
    </hint>
    <hint type="destinationlabel">
     <x>296</x>
     <y>145</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onAccept()</slot>
 </slots>
</ui>
